Testreglar/1.1.1/Nett/nett-1.1.1a.json

```diff
@@ -0,0 +1,506 @@
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		},
+		{
+			"title": "3.8"
+		},
+		{
+			"title": "3.9"
+		},
+		{
+			"title": "3.10"
+		},
+		{
+			"title": "3.11"
+		},
+		{
+			"title": "3.12"
+		},
+		{
+			"title": "3.13"
+		},
+		{
+			"title": "3.14"
+		},
+		{
+			"title": "3.15"
+		},
+		{
+			"title": "3.16"
+		},
+		{
+			"title": "3.17"
+		},
+		{
+			"title": "3.18"
+		},
+		{
+			"title": "3.19"
+		},
+		{
+			"title": "3.20"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Finnes det ikke-lenkede bilder på testsiden?",
+			"ht": "<p>Slik finner du bilder:</p>\n<ul>\n<li>Du kan bruke verktøyet <a href=\"https://chrome.google.com/webstore/detail/web-developer/bfbameneiokkgbdmiekhjnmfkcnldhhm\" target=\"_blank\" rel=\"noopener\">Web Developer</a> i Chrome på testsiden</li>\n<li>Gå til fane \"Images\"</li>\n<li>Velg \"Outline All Images\", \"Outline Background Images\" og \"Display Alt Attributtes\"\n<ul>\n<li>Alle bilder på siden blir markert med en rød ramme</li>\n</ul>\n</li>\n</ul>\n<p><strong>Merk:</strong></p>\n<ul>\n<li>Du skal bare teste bilder som ikke er lenket verken i en lenke eller knapp.</li>\n<li>Verktøyet avdekker ikke bilder kodet med elementer <code>&lt;figure&gt;, &lt;svg&gt;, &lt;picture&gt;, &lt;canvas&gt;</code>. Slike bilder skal testes i steg 2.3 ved velg \"Nei\" i dette steget.</li>\n</ul>\n<p>Bilde kan for eksempel være:</p>\n<ul>\n<li>illustrasjoner</li>\n<li>pyntebilde og dekor</li>\n<li>grafer og diagram</li>\n<li>ikon og symbol</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Finnes det ikke-lenkede bilder kodet med &lt;figure&gt;, &lt;svg&gt; eller &lt;canvas&gt; på testsiden?",
```

Comment:
Har vi vurdert om SVG bør testes for seg?  ref https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html

Reply:
Ja, vi bruker Accessibility Checker som sjekker tilgjengelig navn uavhengig av elementtype.